src/code-templates/libraries/logger/logger.ts

```diff
@@ -1,10 +1,16 @@
-import { configurationProvider } from "configuration-provider";
+import {configurationProvider} from "../configuration-provider";
+import {pino} from "pino";
 
-// example on how to read the level: configurationProvider.get("logger.level")
-export const info = (message) => {
-  console.log(message);
-};
 
-export const error = (message) => {
-  console.error(message);
-};
+const logFile = configurationProvider.get("logger.destination");
```

Comment:
I believe we should always log to stdout and leave the routing decision to 'ops'. Why? I tried to explain here but always open for new ideas

![image](https://user-images.githubusercontent.com/8571500/167805434-88ca3a5a-713d-4dd7-825b-155be625f445.png)

https://github.com/goldbergyoni/nodebestpractices